applications/asset_tracker/src/main.c

```diff
@@ -465,23 +474,55 @@ static void send_cell_pos_request(struct k_work *work)
 
 	ARG_UNUSED(work);
 
+#if !defined(CONFIG_CELL_POS_MULTICELL)
```

Comment:
can you add a `lte_lc_cells_info * data = NULL;` and if mcell is enabled set `data = &cell_data`?
the pointer can be passed to `nrf_cloud_cell_pos_request` so the duplicate call and associated comment can be removed.

Reply:
I had considered doing so.  Thanks for the nudge.